frontend/src/components/sidebar/Sidebar.jsx

```diff
@@ -160,9 +160,8 @@ const Sidebar = () => {
     <>
       <aside
         id='sidebar'
-        className={`fixed top-0 left-0 z-30 w-48 h-screen bg-gray-800 text-white flex flex-col transition-transform duration-300 ease-in-out ${
-          isSidebarOpen ? 'translate-x-0' : '-translate-x-full'
-        } md:relative md:translate-x-0`}
+        className={`fixed top-0 left-0 z-30 w-48 h-screen bg-gray-800 text-white flex flex-col transition-transform duration-300 ease-in-out ${isSidebarOpen ? 'translate-x-0' : '-translate-x-full'
+          } md:relative md:translate-x-0`}
```

Comment:
Why was a change in the sidebar logic needed? Is this a separate fix that you noticed or was this related to WSL rendering it differently?

Reply:
This must've been an auto lint I overlooked. I'll revert it then merge.